core/src/main/java/com/scalar/db/common/ActiveTransactionManagedDistributedTransactionManager.java

```diff
@@ -127,11 +134,28 @@ public synchronized void delete(Delete delete) throws CrudException {
       super.delete(delete);
     }
 
+    /** @deprecated As of release 3.13.0. Will be removed in release 5.0.0. */
+    @Deprecated
     @Override
     public synchronized void delete(List<Delete> deletes) throws CrudException {
       super.delete(deletes);
     }
 
+    @Override
+    public void insert(Insert insert) throws CrudException {
+      super.insert(insert);
```

Comment:
[trivial] This can be removed?

Reply:
I forgot to add `synchronized` to this method, and I've added it in ad802e94063f32ce5cad5089b44920eed22ea890. To add `synchronized`, I think we need this method.